### DISCUSSION TITLE
DNN/Vanjaro tabs on DNN login page

### DISCUSSION THREAD

**Discussion by eswanzey** [vanjarosoftware/Vanjaro.Platform#1183] [Q&A]
In a hybrid site (DNN & Vanjaro), the login screen on a DNN page has two tabs labelled Standard and Vanjaro. Is it possible to disable the Vanjaro tab since it is redundant? Kind of the reverse of what happens when logging into a Vanjaro page - all you see is the "Vanjaro login."

**Comment by eswanzey** [comment]
Here's some css to hide the tabs on DNN

`/*hide login tabs*/
.dnnLogin .LoginTabGroup span {display:none;}`

**Comment by meetmandeep** [comment]
> In a hybrid site (DNN & Vanjaro), the login screen on a DNN page has two tabs labelled Standard and Vanjaro.

Vanjaro will automatically create a Sign In page if one was not explicitly set on DNN. It's recommended that you use Vanjaro page for Sign In functionality since Vanjaro includes its own Authentication Provider that is not WebForms driven. 

> Is it possible to disable the Vanjaro tab since it is redundant?

No. For this reason, you should consider using Vanjaro Page as a Sign In page. If you must use DNN page then your CSS hack works well.

> Kind of the reverse of what happens when logging into a Vanjaro page - all you see is the "Vanjaro login."

Because Vanjaro does not use any of the WebForms functionality so we leave the stock provider out.

**Reply by eswanzey** [reply]
I have a Vanjaro for DNN site, with a Vanjaro page that was converted from a DNN login page. Trouble is, it still displays two tabs - Vanjaro and Standard. Since it's a Vanjaro page with a Vanjaro sign-in block, I expected the "Standard" tab to automagically disappear but it didn't. Searching through settings didn't reveal anything to alter that behavior. Am I missing something?

**Reply by eswanzey** [reply]
Another interesting thing after the conversion is that I can now only login via username. Login via email address no longer works.  The site is set to "use email address as username" and it was working fine until converting the login page to Vanjaro.
